### DISCUSSION TITLE
Do we Have any course available where i can learn step by step

### DISCUSSION THREAD

**Discussion by amitmiglani91** [strimzi/strimzi-kafka-operator#8008] [Q&A]
Hi, i need to understand Strimzi in Depth, i tried with document but having issue, do we have any courses ( including Paid) , please let me know

**Comment by scholzj** [comment]
There are many materials available in our docs, YouTube channel, in blog posts on the website. But I'm not aware of any courses or training like that which would be specifically about Strimzi.

**Reply by amitmiglani91** [reply]
@scholzj Thanx for reply
